src/basic_operations.jl

```diff
@@ -18,6 +18,22 @@
 @inline Base.:*(S::AbstractTensor, n::Number) = _map(x -> x*n, S)
 @inline Base.:*(n::Number, S::AbstractTensor) = _map(x -> n*x, S)
 @inline Base.:/(S::AbstractTensor, n::Number) = _map(x -> x/n, S)
+function Base.:^(S::SecondOrderTensor, p::Int)
+    if p == 1
+        return S
+    elseif p == 0
+        return one(S)
+    elseif p == -1
+        return inv(S)
+    elseif p < 0
```

Comment:
The errors are following:

constructors: Test Failed
  Expression: eltype(tens_arr1) == (Tensors.Tensor){1, 1, Float64, N}
   Evaluated: Tensors.Tensor{1,1,T,M} where M where T<:Real == Tensors.Tensor{1,1,Float64,1}